pkg/vm/vm.go

```diff
@@ -236,6 +236,73 @@ func (v *VM) Step() {
 	}
 }
 
+// StepInto an action to take in the debugger. If the line does not contain a function it behaves the same as
+// “step over” but if it does the debugger will enter the called function and continue line-by-line debugging there.
+func (v *VM) StepInto() {
+	ctx := v.Context()
+
+	if v.istack.Len() == 0 {
+		v.state |= haltState
+	}
+
+	if v.state.HasFlag(haltState) || v.state.HasFlag(faultState) {
```

Comment:
We have `v.HasFailed()` and maybe it makes sense to also add `v.HasStopped()`.

Reply:
ok sure!